spec/support/factories.rb

```diff
@@ -4,7 +4,22 @@
     password "password"
   end
 
+  factory :item do
+    name
+    description "test description"
+    price "5.99"
+    image "http://i.imgur.com/kgOqHMk.gif"
+  end
+
+  factory :category do
+    name
+  end
+
   sequence :username do |n|
     "User #{n}"
   end
+
+  sequence :name do |n|
+    "Name-#{n}"
```

Comment:
Currently went for name n to describe the field it was generating. Its also used for categories, though regardless, might not be bad to change and just use a different category name generator.